src/main/java/taxi/dao/jdbc/DriverJdbcDao.java

```diff
@@ -0,0 +1,109 @@
+package taxi.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import taxi.dao.DriverDao;
+import taxi.lib.Dao;
+import taxi.model.Driver;
+import taxi.services.util.ConnectionUtil;
+
+@Dao
+public class DriverJdbcDao implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String addDriverQuery = "INSERT INTO drivers (name, licence) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(addDriverQuery,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            preparedStatement.setString(1, driver.getName());
+            preparedStatement.setString(2, driver.getLicenceNumber());
+            preparedStatement.executeUpdate();
+            ResultSet resultSet = preparedStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                driver.setId(resultSet.getObject(1, Long.class));
+            }
+            return driver;
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't add driver" + driver, e);
+        }
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        String getByIdQuery = "SELECT * FROM drivers WHERE id = ? AND deleted = false";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(getByIdQuery)) {
+            preparedStatement.setLong(1, id);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(getDriver(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't get driver by id " + id, e);
+        }
+        throw new RuntimeException("Can't get driver by id " + id);
```

Comment:
```suggestion
        return Optional.empty();
```
you will either return Optional.of() or Optional.empty()